<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditerDialog</class>
 <widget class="QDialog" name="MapEditerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="GameBoard" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;マップを作成する画面です。クリックおよびドラッグで配置することができます。&lt;/p&gt;&lt;p&gt;COOLとHOTは各１つのみ配置できます。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;マップに使用する部品を選択します。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="ObjectCounter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>100</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TURN : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="TurnSpin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>マップのターン数です。</string>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="SymmetryCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>チェックを入れた場合、マップは点対称になります。</string>
       </property>
       <property name="text">
        <string>対称</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="whatsThis">
        <string>マップの大きさを設定します。</string>
       </property>
       <item>
        <property name="text">
         <string>決戦(15x17)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>広域(21x17)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="randomGenerateButton">
       <property name="text">
        <string>ランダム生成</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>全てのアイテム、ブロックを取り除きます。</string>
       </property>
       <property name="text">
        <string>全消し</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="WriteButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>マップを書き出します。</string>
       </property>
       <property name="text">
        <string>書き出し</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;マップエディタを完了し、閉じます。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>完了</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameBoard</class>
   <extends>QWidget</extends>
   <header>GameBoard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MapEditerDialog</receiver>
   <slot>Clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MapEditerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WriteButton</sender>
   <signal>clicked()</signal>
   <receiver>MapEditerDialog</receiver>
   <slot>Export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MapEditerDialog</receiver>
   <slot>ComboChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Clear()</slot>
  <slot>Export()</slot>
  <slot>ComboChanged(QString)</slot>
 </slots>
</ui>
